target/linux/realtek/dts-5.10/rtl931x.dtsi

```diff
@@ -0,0 +1,175 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include <dt-bindings/interrupt-controller/mips-gic.h>
+
+/ {
+	#address-cells = <1>;
+	#size-cells = <1>;
+	
+	compatible = "realtek,rtl838x-soc";
+
+	cpus {
+		#address-cells = <1>;
+		#size-cells = <0>;
+		frequency = <1000000000>;
+
+		cpu@0 {
+			// try: 
+			compatible = "mti,interaptive";
+			reg = <0>;
+		};
+
+		cpu@1 {
+			// try: 
+			compatible = "mti,interaptive";
+			reg = <0>;
```

Comment:
```suggestion
reg = <1>;
```
Would be weird if both cores have ID 0...

Reply:
Thanks! Fixed.